<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TestInterface</class>
 <widget class="QWidget" name="TestInterface">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>757</width>
    <height>637</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="CardWidget" name="headerCard">
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="IconWidget" name="testIcon">
          <property name="minimumSize">
           <size>
            <width>16</width>
            <height>16</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16</width>
            <height>16</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="BodyLabel" name="testNameLabel">
          <property name="text">
           <string>TESTNAME</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="CardWidget" name="CardWidget_5">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>0</height>
         </size>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_9">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_10">
           <item>
            <widget class="IconWidget" name="userNameIcon">
             <property name="minimumSize">
              <size>
               <width>16</width>
               <height>16</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>16</width>
               <height>16</height>
              </size>
             </property>
            </widget>
           </item>
           <item>
            <widget class="BodyLabel" name="userNameLabel">
             <property name="text">
              <string>NAME</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="CardWidget" name="timeCard">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_8">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_7">
           <property name="spacing">
            <number>7</number>
           </property>
           <property name="leftMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="IconWidget" name="timeIcon">
             <property name="minimumSize">
              <size>
               <width>16</width>
               <height>16</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>16</width>
               <height>16</height>
              </size>
             </property>
            </widget>
           </item>
           <item>
            <widget class="BodyLabel" name="timeLabel">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>65</width>
               <height>0</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>65</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="text">
              <string>44:44:44</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="CardWidget" name="bodyCard">
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <layout class="QVBoxLayout" name="verticalLayout_9">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <property name="spacing">
           <number>0</number>
          </property>
          <item>
           <widget class="SmoothScrollArea" name="SmoothScrollArea">
            <property name="styleSheet">
             <string notr="true">background-color: transparent;
border: 0;</string>
            </property>
            <property name="widgetResizable">
             <bool>true</bool>
            </property>
            <widget class="QWidget" name="scrollAreaWidgetContents">
             <property name="geometry">
              <rect>
               <x>0</x>
               <y>0</y>
               <width>717</width>
               <height>229</height>
              </rect>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_5">
              <property name="spacing">
               <number>0</number>
              </property>
              <property name="leftMargin">
               <number>0</number>
              </property>
              <property name="topMargin">
               <number>0</number>
              </property>
              <property name="rightMargin">
               <number>0</number>
              </property>
              <property name="bottomMargin">
               <number>0</number>
              </property>
              <item>
               <layout class="QVBoxLayout" name="verticalLayout_3">
                <property name="spacing">
                 <number>9</number>
                </property>
                <property name="bottomMargin">
                 <number>0</number>
                </property>
                <item>
                 <widget class="BodyLabel" name="questionTitleLabel">
                  <property name="text">
                   <string>text</string>
                  </property>
                  <property name="wordWrap">
                   <bool>true</bool>
                  </property>
                  <property name="textInteractionFlags">
                   <set>Qt::NoTextInteraction</set>
                  </property>
                  <property name="buddy">
                   <cstring></cstring>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="HorizontalSeparator" name="HorizontalSeparator"/>
                </item>
               </layout>
              </item>
              <item>
               <widget class="SmoothScrollArea" name="SmoothScrollArea_2">
                <property name="widgetResizable">
                 <bool>true</bool>
                </property>
                <widget class="QWidget" name="answerBox">
                 <property name="geometry">
                  <rect>
                   <x>0</x>
                   <y>0</y>
                   <width>717</width>
                   <height>196</height>
                  </rect>
                 </property>
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <layout class="QVBoxLayout" name="verticalLayout_4">
                  <property name="spacing">
                   <number>12</number>
                  </property>
                  <property name="leftMargin">
                   <number>0</number>
                  </property>
                  <property name="topMargin">
                   <number>12</number>
                  </property>
                  <property name="rightMargin">
                   <number>12</number>
                  </property>
                  <property name="bottomMargin">
                   <number>12</number>
                  </property>
                  <item>
                   <widget class="CheckBox" name="CheckBox">
                    <property name="text">
                     <string>Check box</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="RadioButton" name="RadioButton">
                    <property name="text">
                     <string>Radio button</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="LineEdit" name="LineEdit">
                    <property name="text">
                     <string>test</string>
                    </property>
                    <property name="dragEnabled">
                     <bool>false</bool>
                    </property>
                    <property name="placeholderText">
                     <string>OTVER</string>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </widget>
               </widget>
              </item>
             </layout>
            </widget>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item row="1" column="0">
       <widget class="ProgressBar" name="ProgressBar">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>4</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>4</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="CardWidget" name="bottomCard">
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <item>
       <widget class="PushButton" name="endButton">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>150</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>END TEST</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="PushButton" name="saveButton">
        <property name="minimumSize">
         <size>
          <width>150</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>150</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>save</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_4">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <property name="spacing">
         <number>6</number>
        </property>
        <item>
         <widget class="ToolButton" name="prevButton">
          <property name="toolButtonStyle">
           <enum>Qt::ToolButtonIconOnly</enum>
          </property>
          <property name="autoRaise">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="ToolButton" name="nextButton">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CheckBox</class>
   <extends>QCheckBox</extends>
   <header>qfluentwidgets</header>
  </customwidget>
  <customwidget>
   <class>PushButton</class>
   <extends>QPushButton</extends>
   <header>qfluentwidgets</header>
  </customwidget>
  <customwidget>
   <class>ToolButton</class>
   <extends>QToolButton</extends>
   <header>qfluentwidgets</header>
  </customwidget>
  <customwidget>
   <class>RadioButton</class>
   <extends>QRadioButton</extends>
   <header>qfluentwidgets</header>
  </customwidget>
  <customwidget>
   <class>IconWidget</class>
   <extends>QWidget</extends>
   <header>qfluentwidgets</header>
  </customwidget>
  <customwidget>
   <class>HorizontalSeparator</class>
   <extends>QWidget</extends>
   <header>qfluentwidgets</header>
  </customwidget>
  <customwidget>
   <class>CardWidget</class>
   <extends>QFrame</extends>
   <header>qfluentwidgets</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>SmoothScrollArea</class>
   <extends>QScrollArea</extends>
   <header>qfluentwidgets</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>BodyLabel</class>
   <extends>QLabel</extends>
   <header>qfluentwidgets</header>
  </customwidget>
  <customwidget>
   <class>ProgressBar</class>
   <extends>QProgressBar</extends>
   <header>qfluentwidgets</header>
  </customwidget>
  <customwidget>
   <class>LineEdit</class>
   <extends>QLineEdit</extends>
   <header>qfluentwidgets</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../resource/resource.qrc"/>
 </resources>
 <connections/>
</ui>
